app/build.gradle

```diff
@@ -34,9 +34,6 @@ publishing {
             groupId packageName
             version = libraryVersion
             artifactId project.getName()
-
-            // Tell maven to prepare the generated "*.aar" file for publishing
-            artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
```

Comment:
not sure if I should removed this

Reply:
Please, revert this change, it's about publishing the storage libary itself)